<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Module</class>
 <widget class="QWidget" name="Module">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KWin script configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="messageWidget" native="true"/>
   </item>
   <item>
    <widget class="KPluginSelector" name="scriptSelector" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        </sizepolicy>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importScriptButton">
       <property name="text">
        <string>Install from File...</string>
       </property>
       <property name="icon">
        <iconset theme="document-import">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KNS3::Button" name="ghnsButton">
       <property name="text">
        <string>Get New Scripts...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPluginSelector</class>
   <extends>QWidget</extends>
   <header>kpluginselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KNS3::Button</class>
   <extends>QPushButton</extends>
   <header>KNS3/Button</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
